test/libsolidity/syntaxTests/using/using_contract_err.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+    function f(uint) external {
+    }
+}
+interface I {
+    function f(uint) external;
+}
+
+contract Test {
+    using C for uint;
+}
+// ----
+// TypeError 4357: (127-128): Library name expected. If you want to attach a function, use '{...}'.
```

Comment:
If we want to be perfectionistic, we can not emit the second part inside a contract... but also fine.